ceph/exporter.go

```diff
@@ -212,8 +213,8 @@ func (exporter *Exporter) Describe(ch chan<- *prometheus.Desc) {
 		exporter.Logger.WithError(err).Error("failed to set rbd mirror")
 		return
 	}
-
-	for _, cc := range exporter.getCollectors() {
+	exporter.initCollectors()
```

Comment:
`Describe()` can be called multiple times, I think. Can we call this in `NewExporter()` instead?